### PR TITLE
remove the message in bottom

### PR DESCRIPTION
**Pull request by RachaelDing**
https://nanopay.atlassian.net/browse/NP-2286
remove the message shown in bottom in nature code review screen

### CODE REVIEW

**Review by jlhughes — CHANGES_REQUESTED** [review]
@lchanmann @RachaelDing  Rachael where is this being called/setup from?  That is where this should be set to false.

**Comment by RachaelDing** [comment]
@jlhughes 
This is only called in the initE() method of the same class. And nowhere setup it after initialization, so I changed the initial value of if. Should I set it in iniE() method instead? Thank you.

**Comment by lchanmann** [comment]
@RachaelDing there are two places that call it and is bound to this.isValid$
Screen Shot 2020-10-20 at 12 02 47 PM

**Comment by RachaelDing** [comment]
@lchanmann  I see what you meaned... Thank you

File: src/foam/nanos/crunch/ui/CapableMinMaxCapabilityWizardlet.js [hunk 1]
```diff
@@ -182,6 +183,7 @@ foam.CLASS({
         choices$: this.choices$,
         booleanView: this.CardSelectView,
         isValidNumberOfChoices$: this.isValid$,
+        showValidNumberOfChoicesHelper: false,
```
Comment (jlhughes) [inline src/foam/nanos/crunch/ui/CapableMinMaxCapabilityWizardlet.js]: remove this please.

Comment (RachaelDing) [reply src/foam/nanos/crunch/ui/CapableMinMaxCapabilityWizardlet.js]: removed

**Pull request merged**